File: Todo.WebApi/Todo.WebApi/Models/Todo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Todos.WebApi.Models
{
  public class Todo
  {
    public string Id { get; set; }

    public string Description { get; set; }

    public bool Completed { get; set; }

    public int CategoryId { get; set; }
  }

  public class Category
  {
    public int Id { get; set; }
    public string Description { get; set; }

    public int Count { get; set; }
  }

}

File: src/app/modals/Todo.ts
export class Todo {

  constructor(private descr: string, private done: boolean, private show: boolean, categoryId: number) {
    this.Description = descr;
    this.Completed = done;
    this.Show = show;
    this.CategoryId = categoryId;
  }

  Description: string;
  Completed: boolean;
  Show: boolean;
  Id: string;
  CategoryId: number = 1;
}

export class Category {
  Id: number;
  Description: string;
  Count: number;
}